src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/SwitchServiceImpl.java

```diff
@@ -274,7 +272,10 @@ public CompletableFuture<SwitchValidationResult> validateSwitch(SwitchId switchI
         logger.info("Validate request for switch {}", switchId);
 
         return performValidateV2(
-                SwitchValidateRequest.builder().switchId(switchId).processMeters(true).build())
+                SwitchValidateRequest.builder()
```

Comment:
`SwitchValidateRequest` produced not only by NB, but by "network" and "nbworker" (perhaps some others) components too. Is their requests still works as expected?

Reply:
fixed